l10n_nl_tax_statement_icp/models/l10n_nl_vat_statement_icp_line.py

```diff
@@ -32,6 +35,14 @@ class VatStatementIcpLine(models.Model):
     amount_services = fields.Monetary(readonly=True)
     format_amount_services = fields.Char(compute="_compute_icp_amount_format")
 
+    @api.depends("vat", "country_code")
+    def _compute_format_vat(self):
+        for line in self:
+            if line.country_code and line.vat:
+                line.format_vat = line.vat.lstrip(line.country_code)
+            else:
+                line.format_vat = line.format_vat
```

Comment:
Should this not be: `line.format_vat = line.vat` ?

Reply:
I just wanted to return the same value, but maybe you are right.. is better that way